src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1898,6 +1904,34 @@ private void AddNodeTypeToSearch(TypeLoadData typeLoadData)
             SearchModel.Add(new NodeModelSearchElement(typeLoadData));
         }
 
+        /// <summary>
+        /// This method updates the node search library to either hide or unhide nodes that belong
+        /// to a specified assembly name and namespace. These nodes will be hidden from the node
+        /// library sidebar and from the node search.
+        /// </summary>
+        /// <param name="hide">Set to true to hide, set to false to unhide.</param>
+        /// <param name="library">The assembly name of the library.</param>
+        /// <param name="namespc">The namespace of the nodes to be hidden.</param>
+        internal void HideUnhideNamespace(bool hide, string library, string namespc)
```

Comment:
Ideally this could be made a public method that is exposed thro' the `IViewExtension` interface for 3rd party developers to add their own namespaces that they might want to hide or show in a similar fashion like we're doing for T-splines but that need might be too far out for now this is okay.